Trap Quest.inform/Inform/Extensions/Furniture/Hotel Bed.i7x

```diff
@@ -15,7 +15,15 @@ To compute furniture resting on (F - a hotel bed):
 To say RestingDesc of (F - a hotel bed):
 	say "You [one of]get into the bed and [or]continue to [stopping]relax on the comfortable mattress.".
 
-HotelBedBoring is a kind of hotel bed.  There are 2 HotelBedBoring.
+HotelBedHalfPosted is a kind of hotel bed. The description is "A bed with four posts. Two high headboard posts and two footboard posts[if the sex addiction of the player > 9] exactly the right height[end if].".
```

Comment:
The idea with the hotel beds is that until they are slept in for the first time they all have the same description when they are examined, so that the player doesn't know what effect they are about to trigger.  Replacing the boring hotel beds with these breaks that mechanic, as now there's a way to determine which bed is which without sleeping on it.  I'm not sure I'm happy about that.

Reply:
Ah, I see. We can then either extend it each of the 4 hotel beds, save it for a different kind of bed, or just leave it the way it was.